drivers/gpio/Kconfig.nrf5

```diff
@@ -27,13 +25,27 @@ config GPIO_NRF5_P0_DEV_NAME
 	help
 	  Specify the device name to be used for the GPIO port.
 
-config GPIO_NRF5_PORT_P0_PRI
-	int "GPIOTE P0 interrupt priority"
-	depends on GPIO_NRF5_P0
-	range 0 1 if SOC_SERIES_NRF51X
+config GPIO_NRF5_P1
+	bool "nRF5x GPIO Port P1"
+	depends on SOC_NRF52840
+	help
+	  Enable nRF5 GPIO port P1 config options.
+
+config GPIO_NRF5_P1_DEV_NAME
+	string "GPIO Port P1 Device Name"
+	depends on GPIO_NRF5_P1
+	default "GPIO_1"
+	help
+	  Specify the device name to be used for the GPIO port.
+
+config GPIOTE_NRF5_PRI
+	int "GPIOTE interrupt priority"
+	depends on GPIO_NRF5_P0 || GPIO_NRF5_P1
+	range 0 2 if SOC_SERIES_NRF51X
 	range 0 5 if SOC_SERIES_NRF52X
-	default 1
+	default 2 if SOC_SERIES_NRF51X
```

Comment:
Changing the default interrupt priority (1->2 on nRF51 and 1->5 on nRF52) almost seems like a patch that should land on top of the GPIO driver change to support P1.   It might go unnoticed as a part of this larger patch.  What do you think?

Reply:
I will make it a separate commit.